depsync/service.go

```diff
@@ -0,0 +1,22 @@
+package depsync
+
+import (
+	"context"
+)
+
+type Service interface {
+	SyncNow(ctx context.Context)
+}
+
+type syncNowService struct {
+	syncer Syncer
+}
+
+func (s *syncNowService) SyncNow(_ context.Context) {
+	s.syncer.SyncNow()
+	return
+}
+
+func NewRPC(syncer Syncer) *syncNowService {
```

Comment:
Return an exported service, otherwise the types are hard to deal with.

Reply:
I modeled this on other existing services. Happy to change it.